src/shapes/SurfaceRectangle.js

```diff
@@ -192,5 +192,18 @@ define([
 
         };
 
+        // Internal use only. Intentionally not documented.
+        SurfaceRectangle.prototype.getReferencePosition = function () {
+            return this.center;
+        };
+
+        // Internal use only. Intentionally not documented.
+        SurfaceRectangle.prototype.moveTo = function (globe, position) {
+            var locations = [];
+            locations.push(new Location(this.center.latitude, this.center.longitude));
```

Comment:
Could the center property be used directly instead of creating a new location object?